File: Algorithms/119. Pascal's Triangle II/swift/solution.brute.force.swift
class Solution {
    func getRow(_ rowIndex: Int) -> [Int] {
        var ans: [Int] = [];
        for i in 0...rowIndex {
            ans.append(getNum(rowIndex, i))
        }
        return ans
    }

    func getNum(_ row: Int, _ col: Int) -> Int {
        if row == 0 || col == 0 || row == col {
            return 1
        }
        return getNum(row - 1, col - 1) + getNum(row - 1, col)
    }
}

File: Algorithms/119. Pascal's Triangle II/kotlin/solution.brute_force.kt
class Solution {
    fun getRow(rowIndex: Int): List<Int> {
        val ans = ArrayList<Int>()
        for (i in 0..rowIndex) {
            ans.add(getNum(rowIndex, i))
        }
        return ans
    }

    fun getNum(row: Int, col: Int): Int {
        if (row == 0 || col == 0 || row == col) {
            return 1
        }
        return getNum(row - 1, col - 1) + getNum(row - 1, col)
    }
}